docs/cmake-project-view.md

```diff
@@ -0,0 +1,124 @@
+
+Users can specify what is shown in the status bar through their `settings.json` file.
+
+The default settings will be set to the following:
+## Default Settings Json
+```json
+"cmake.options.statusBarVisibility": "hidden"
+"cmake.options.advanced": {
+    "build": {
+        "statusBarVisibility": "visible"
+    },
+    "launch": {
+        "statusBarVisibility": "visible"
+    },
+    "debug": {
+        "statusBarVisibility": "visible"
+    }
+}
+```
+
+These settings mean that by default all settings will be hidden from the status bar, except for Build, Launch, and Debug actions.
+
+To revert to the prior experience and have all of your presets display in the status bar, set your `Cmake > Options: Status Bar: Visibility` setting to `Visible`. This setting is overwritten by`CMake > Options: Advanced`, so if you had added to have some options hidden through this, these options will still remain hidden. 
+
+![Screenshot of the Visual Studio Code Settings view, with the CMake Status Bar: Visibility options. You can set these to visible, hidden, compact, or icon](images/cmake-statusbar-setting.png)
+
+## Configuring your CMake Status Bar
+
+ You can configure settings for each of the following CMake actions in your settings.json to either be `visible` or `hidden` in the status bar through the `statusBarVisibility` variable.  Settings that are available to be hidden from the project status view side bar can be configured to be `visible` or `hidden` through the `projectStatusVisibility` variable.
+ 
+To make options visible in the status bar take up less space, you can configure certain options to be only the respective icon through the `icon` option in `statusBarVisibility` or specify a given character length through the `compact` option in `statusBarVisibility`. If a `statusBarVisibility` option is set to `compact` you can then specify an integer length for how many characters you want an option to take up through the `statusBarLength` option. It will truncate your existing status bar option to that specified character length. The default option is 20 characters.
+
+You can also configure options to be `visible` or `hidden` in the Project Status View in the CMake Tools sidebar. The options that allow for this customization are:
+*`folder`
+*`configure`
+*`build`
+*`ctest`
+*`debug`
+*`launch`
+Note: if you set one of these to hidden, the parent node will entirely be hidden. For example, if you set `build` to `hidden` in `projectStatusVisibility`, the option to select your build preset will also be hidden
+ 
+The full level of options for the CMake status can be seen below:
+
+### CMake Status Bar Configuration Options in your Settings Json
```

Comment:
```suggestion
### CMake Status Bar & Project Status View Configuration Options in your Settings Json
```

Reply:
you can ignore this one. just an idea

Reply:
```suggestion
### CMake Status Bar and Project Status View Configuration Options in your Settings Json
```

I like this suggestion but I think (I could be wrong) that the common practice is to spell out the word "and".